frontend/src/pages/index.js

```diff
@@ -22,8 +25,23 @@ type Props = {
   result: {}
 }
 
+function mapStateToProps(state) {
+  return {
+    candidates: state.candidates
+  }
+}
```

Comment:
some cool syntax you could is:

const mapStateToProps =(state) => (
    {candidates: state.candidates}
)

i think

Reply:
you don’t have to change it to that though